src/test/java/com/wikia/webdriver/testcases/forumtests/ForumEditModeTests.java

```diff
@@ -1,13 +1,17 @@
 package com.wikia.webdriver.testcases.forumtests;
 
+import org.openqa.selenium.remote.server.handler.GetCurrentUrl;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.remote.server.handler.GetCurrentUrl'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)